2022-11-05 13:25:51,882 INFO    MainThread:31495 [wandb_setup.py:_flush():68] Configure stats pid to 31495
2022-11-05 13:25:51,882 INFO    MainThread:31495 [wandb_setup.py:_flush():68] Loading settings from /home/piperunner/.config/wandb/settings
2022-11-05 13:25:51,882 INFO    MainThread:31495 [wandb_setup.py:_flush():68] Loading settings from /home/piperunner/Projects/NTNU/SI/ntnu-si-assignment-3/notebooks/wandb/settings
2022-11-05 13:25:51,883 INFO    MainThread:31495 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True', 'silent': 'true'}
2022-11-05 13:25:51,883 INFO    MainThread:31495 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-11-05 13:25:51,883 INFO    MainThread:31495 [wandb_init.py:_log_setup():476] Logging user logs to /home/piperunner/Projects/NTNU/SI/ntnu-si-assignment-3/notebooks/wandb/run-20221105_132551-1f7tkn6d/logs/debug.log
2022-11-05 13:25:51,883 INFO    MainThread:31495 [wandb_init.py:_log_setup():477] Logging internal logs to /home/piperunner/Projects/NTNU/SI/ntnu-si-assignment-3/notebooks/wandb/run-20221105_132551-1f7tkn6d/logs/debug-internal.log
2022-11-05 13:25:51,883 INFO    MainThread:31495 [wandb_init.py:_jupyter_setup():426] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f8613a82970>
2022-11-05 13:25:51,883 INFO    MainThread:31495 [wandb_init.py:init():516] calling init triggers
2022-11-05 13:25:51,884 INFO    MainThread:31495 [wandb_init.py:init():519] wandb.init called with sweep_config: {}
config: {}
2022-11-05 13:25:51,884 INFO    MainThread:31495 [wandb_init.py:init():569] starting backend
2022-11-05 13:25:51,884 INFO    MainThread:31495 [wandb_init.py:init():573] setting up manager
2022-11-05 13:25:51,888 INFO    MainThread:31495 [backend.py:_multiprocessing_setup():102] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-11-05 13:25:51,891 INFO    MainThread:31495 [wandb_init.py:init():580] backend started and connected
2022-11-05 13:25:51,901 INFO    MainThread:31495 [wandb_run.py:_label_probe_notebook():1113] probe notebook
2022-11-05 13:25:51,901 INFO    MainThread:31495 [wandb_run.py:_label_probe_notebook():1123] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-11-05 13:25:51,901 INFO    MainThread:31495 [wandb_init.py:init():658] updated telemetry
2022-11-05 13:25:51,912 INFO    MainThread:31495 [wandb_init.py:init():693] communicating run to backend with 60 second timeout
2022-11-05 13:25:52,503 INFO    MainThread:31495 [wandb_run.py:_on_init():2000] communicating current version
2022-11-05 13:25:52,606 INFO    MainThread:31495 [wandb_run.py:_on_init():2004] got version response 
2022-11-05 13:25:52,606 INFO    MainThread:31495 [wandb_init.py:init():728] starting run threads in backend
2022-11-05 13:25:53,828 INFO    MainThread:31495 [wandb_run.py:_console_start():1980] atexit reg
2022-11-05 13:25:53,828 INFO    MainThread:31495 [wandb_run.py:_redirect():1838] redirect: SettingsConsole.WRAP_RAW
2022-11-05 13:25:53,828 INFO    MainThread:31495 [wandb_run.py:_redirect():1903] Wrapping output streams.
2022-11-05 13:25:53,828 INFO    MainThread:31495 [wandb_run.py:_redirect():1925] Redirects installed.
2022-11-05 13:25:53,828 INFO    MainThread:31495 [wandb_init.py:init():765] run started, returning control to user process
2022-11-05 13:25:53,834 INFO    MainThread:31495 [wandb_run.py:_config_callback():1160] config_cb None None {'bat_count': 10, 'amp': 0.7, 'freq': 0.5, 'freq_min': 0, 'freq_max': 1, 'epochs': 2000, 'network': [6, 5, 4, 3, 2], 'type': 'BA'}
2022-11-05 13:25:58,136 INFO    MainThread:31495 [wandb_init.py:_pause_backend():389] pausing backend
2022-11-05 13:25:58,136 INFO    MainThread:31495 [jupyter.py:save_ipynb():378] not saving jupyter notebook
2022-11-05 13:25:58,140 INFO    MainThread:31495 [wandb_init.py:_resume_backend():398] resuming backend
2022-11-05 13:25:58,344 INFO    MainThread:31495 [wandb_init.py:_pause_backend():389] pausing backend
2022-11-05 13:25:58,345 INFO    MainThread:31495 [jupyter.py:save_ipynb():378] not saving jupyter notebook
2022-11-05 13:28:19,848 INFO    MainThread:31495 [wandb_init.py:_resume_backend():398] resuming backend
2022-11-05 13:28:19,859 INFO    MainThread:31495 [wandb_setup.py:_flush():68] Configure stats pid to 31495
2022-11-05 13:28:19,859 INFO    MainThread:31495 [wandb_setup.py:_flush():68] Loading settings from /home/piperunner/.config/wandb/settings
2022-11-05 13:28:19,859 INFO    MainThread:31495 [wandb_setup.py:_flush():68] Loading settings from /home/piperunner/Projects/NTNU/SI/ntnu-si-assignment-3/notebooks/wandb/settings
2022-11-05 13:28:19,859 INFO    MainThread:31495 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True', 'silent': 'true'}
2022-11-05 13:28:19,859 INFO    MainThread:31495 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-11-05 13:28:19,859 INFO    MainThread:31495 [wandb_init.py:_log_setup():476] Logging user logs to /home/piperunner/Projects/NTNU/SI/ntnu-si-assignment-3/notebooks/wandb/run-20221105_132819-3nxpt2de/logs/debug.log
2022-11-05 13:28:19,859 INFO    MainThread:31495 [wandb_init.py:_log_setup():477] Logging internal logs to /home/piperunner/Projects/NTNU/SI/ntnu-si-assignment-3/notebooks/wandb/run-20221105_132819-3nxpt2de/logs/debug-internal.log
2022-11-05 13:28:19,859 INFO    MainThread:31495 [wandb_init.py:init():516] calling init triggers
2022-11-05 13:28:19,859 INFO    MainThread:31495 [wandb_init.py:init():519] wandb.init called with sweep_config: {}
config: {}
2022-11-05 13:28:19,860 INFO    MainThread:31495 [wandb_init.py:init():535] re-initializing run, found existing run on stack: 1f7tkn6d
2022-11-05 13:28:19,860 INFO    MainThread:31495 [wandb_run.py:_finish():1746] finishing run piperunner/ntnu-si-assignment-3/1f7tkn6d
2022-11-05 13:28:19,860 INFO    MainThread:31495 [jupyter.py:save_history():448] not saving jupyter history
2022-11-05 13:28:19,860 INFO    MainThread:31495 [jupyter.py:save_ipynb():378] not saving jupyter notebook
2022-11-05 13:28:19,860 INFO    MainThread:31495 [wandb_init.py:_jupyter_teardown():408] cleaning up jupyter logic
2022-11-05 13:28:19,860 INFO    MainThread:31495 [wandb_run.py:_atexit_cleanup():1949] got exitcode: 0
2022-11-05 13:28:19,860 INFO    MainThread:31495 [wandb_run.py:_restore():1932] restore
2022-11-05 13:28:19,860 INFO    MainThread:31495 [wandb_run.py:_restore():1938] restore done
